go/rpcwrap/rpcwrap.go

```diff
@@ -185,3 +185,75 @@ func GetRpcPath(codecName string, auth bool) string {
 	}
 	return path
 }
+
+// ServeCustomRPC serves the given http rpc requests with the provided ServeMux,
```

Comment:
ServeCustomRPC -> ServeHTTPRPC.
Can you run golint and see if there are other new warnings?